examples/platform/silabs/efr32/rs911x/hal/efx32_ncp_host.c

```diff
@@ -0,0 +1,287 @@
+/*******************************************************************************

```

Comment:
isn't that file in wifi_sdk ?

Reply:
Yes we have updated it for SPI multiplexing changes so copied till they add the API for SPI multiplexing